task/06-conditions-n-loops-tasks.js

```diff
@@ -30,7 +30,15 @@
  *
  */
 function getFizzBuzz(num) {
-    throw new Error('Not implemented');
+    if (num % 3 === 0 && num % 5 === 0) {
+        return 'FizzBuzz';
+    } else if (num % 3 === 0) {
+        return 'Fizz';
+    } else if (num % 5 === 0) {
+        return 'Buzz';
+    }
```

Comment:
If we are using early return, we can replace `else if` statements with if:

```suggestion
    if (num % 3 === 0 && num % 5 === 0) {
        return 'FizzBuzz';
    }
    
    if (num % 3 === 0) {
        return 'Fizz';
    }
    
    if (num % 5 === 0) {
        return 'Buzz';
    }
```

Also, if we want to check if the number is a multiply of 3 and 5, we can use following statement:

```
if (num % 15 === 0) { ... }
```